tests/imodels-clients-tests/src/integration/authoring/ChangesetOperations.test.ts

```diff
@@ -342,7 +341,7 @@ describe("[Authoring] ChangesetOperations", () => {
       it(`should should retry changeset download if it fails the first time when downloading changeset ${testCase.label}`, async () => {
         // Arrange
         const fileTransferLog = new FileTransferLog();
-        const azureClientStorage = new AzureClientStorage(new BlockBlobClientWrapperFactory());
+        const azureClientStorage = AuthoringUtilityFunctions.createDefaultClientStorage();
```

Comment:
Rename variable to defaultClientStorage

Reply:
Renamed to clientStorage to be more appropriate